eng/common/TestResources/New-TestResources.ps1

```diff
@@ -866,6 +875,11 @@ The environment file will be named for the test resources template that it was
 generated for. For ARM templates, it will be test-resources.json.env. For
 Bicep templates, test-resources.bicep.env.
 
+.PARAMETER DevopsLogging
+By default, the -CI parameter will print out secrets to logs with Azure Pipelines log
```

Comment:
For printing purposes, just always print them out redacted in CI. Don't see why they need to be printed as long as the appropriate env vars are set in the CI. They do need to be print out manually, though, as noted above.

Reply:
It is better to run the devops "##vso issecret=true" magic logging command like we currently do, so that we can prevent any potential secret leakage in the logs from downstream scripts+tests, since devops will filter every subsequent log message for the secret values. However, printing this line in stress test CI environments just causes the secrets to be dumped out like `##vso[task.setvariable variable=_AZURE_CLIENT_SECRET issecret=true ]somesecret` which leaks them.

Reply:
You explained this against my previous comment as well, so withdrawn. I get the intent now, though the parameter naming might be changed to be less confusing.